### PR TITLE
fixes event processor bugs

### PR DESCRIPTION
**Pull request by fugufish**
a few bugs in the event processor, including one that can cause events to double publish

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# CHANGELOG
+
+##  0.1.1
+* fixes bug where event publishing uses the wrong method name to look  up local events
+* fixes condition where events  may double publish when multiple events are registered
```
Comment (stevebooks) [inline CHANGELOG.md]: double spaces

Comment (fugufish) [reply CHANGELOG.md]: not sure I follow

Comment (stevebooks) [reply CHANGELOG.md]: events[space][space]may

Comment (fugufish) [reply CHANGELOG.md]: ah, there is a  bug in my stoopid keyboard doing that, I'll submit another PR with that fix in there

**Pull request merged**